### PR TITLE
MSI: Update RHODS-RHOAM stage scenarios

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam__v4.14-stage.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      ADDON2_CONFIG: name=managed-api-service;addon-managed-api-service=1;rosa-cli-required=true;notification-email=interop-qe-ms@redhat.com;rosa=true
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "addon-install", "failure_type": "all", "classification": "Provisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-380"},
+          {"step": "addon-uninstall", "failure_type": "all", "classification": "Deprovisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-380"},
+          {"step": "addon-tests-*", "failure_type": "all", "classification": "Addons tests", "jira_project": "RHODS", "jira_epic": "RHODS-7870"},
```
Comment (rnetser) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam__v4.14-stage.yaml:44]: please update changes according to https://github.com/openshift/release/pull/42304/ and  https://github.com/openshift/release/pull/42106

Comment (oharan2) [reply ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-rhoam__v4.14-stage.yaml]: Done

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-rhoam-v4.14-stage-co-exist-rosa-rhods-rhoam

**Review by rnetser — CHANGES_REQUESTED** [review]
please rebase

**Pull request merged**